windows/src/developer/kmcomp/main.pas

```diff
@@ -176,6 +177,11 @@ procedure Run;
       Inc(i);
       FParamSourcePath := ParamStr(i);
     end
+    else if s = '-addhelppath' then
```

Comment:
Let's use `-add-help-path` to match the style of other parameters

Reply:
ack, I realized in api.keyman.com, we used `helpLink`. Now to s/helpPath/helpLink/g and `-add-help-link` accordingly